JS/Lesson24/goksel/index.html

```diff
@@ -0,0 +1,51 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Newsletter sign-up form</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css">
+</head>
+
+<body>
+    <div class="form form-container active-form">
+        <form id="myForm" novalidate>
+            <div class="left-container">
+                <h1>Stay updated!</h1>
+                <span>Join 60,000+ product managers receiving monthly updates on:</span>
+                <ul>
+
+                    <li><i class="fa-solid fa-circle-check"></i> Product discovery and building what matters</li>
```

Comment:
In cases like this one, you usually expect a space between an icon and text but you don't want the icon and text be on different lines. For that, you can use non-breaking space character instead of a normal space.
```suggestion
                    <li><i class="fa-solid fa-circle-check"></i>&nbsp;Product discovery and building what matters</li>
```
https://www.w3schools.com/html/html_entities.asp#:~:text=A%20commonly%20used%20HTML%20entity,break%20into%20a%20new%20line).